release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,40 @@
+### **Version 1.2.0.0 Release Notes**
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.0
+
+### **Bug Fixes**
+* Fix quoting and url-encoding ([#153](https://github.com/opensearch-project/dashboards-reports/pull/153))
+* Remove hard coded localhost when calling API ([#172](https://github.com/opensearch-project/dashboards-reports/pull/172))
+* rename plugin helper config file name to consistent with OSD ([#180](https://github.com/opensearch-project/dashboards-reports/pull/180))
+* Support range filters for csv reports ([#185](https://github.com/opensearch-project/dashboards-reports/pull/185))
+* Fix Report Creation after Report Definition Creation ([#196](https://github.com/opensearch-project/dashboards-reports/pull/196))
+* fix csv missing fields issue and empty csv on _source fields ([#206](https://github.com/opensearch-project/dashboards-reports/pull/206))
+
+
+### **Infrastructure**
+* Taking RBAC settings from Alerting plugin default to false ([#165](https://github.com/opensearch-project/dashboards-reports/pull/165))
+* Integrate notifications backend ([#129](https://github.com/opensearch-project/dashboards-reports/pull/129))
+* Build email message from template with reports links ([#184](https://github.com/opensearch-project/dashboards-reports/pull/184))
+* Use advanced settings for date format in csv reports ([#186](https://github.com/opensearch-project/dashboards-reports/pull/186))
+* Remove Notifications from Create/Edit ([#212](https://github.com/opensearch-project/dashboards-reports/pull/212))
+* Remove calling notifications in reports scheduler ([#211](https://github.com/opensearch-project/dashboards-reports/pull/211))
+
+
+### **Enhancements**
+* Add metrics for notifications ([#173](https://github.com/opensearch-project/dashboards-reports/pull/173))
+* Add logic to build report detail page link and send as part of message for non-email channels ([#182](https://github.com/opensearch-project/dashboards-reports/pull/182))
+* Forward all cookies while using headless Chromium ([#194](https://github.com/opensearch-project/dashboards-reports/pull/194))
+* Catch Notifications Errors on Details Pages ([#197](https://github.com/opensearch-project/dashboards-reports/pull/197))
+* Remove notifications integration from Details pages ([#210](https://github.com/opensearch-project/dashboards-reports/pull/210))
```

Comment:
How come some of the PRs for removing notifications are in Infrastructure and others are in Enhancements? I feel they should all be in the same section

Reply:
Shall i put all PRs for removing notifications under Infrastructure ?

Reply:
I would say it should be under Infrastructure, thoughts @joshuali925 ?

Reply:
we use infra to categorize CI/CD, pipeline, UT/IT changes. Not very sure what goes for removing notifications, maybe `Refactoring`? but it doesn't matter as much since notifications was added and removed, no changes for existing users